GUI/UI/UITransition.cpp

```diff
@@ -124,7 +125,9 @@ GUIWindow_Transition::GUIWindow_Transition(uint anim_id, uint exit_pic_id,
                 uCurrentHouse_Animation =
                     IndoorLocation::GetLocationIndex(pLocationName);
         } else {
-            transition_button_label = localization->GetString(LSTR_ENTER);
+            transition_button_label = localization->FormatString(
```

Comment:
Is this the correct behaviour for else case where GetMapInfo(v15) returns 0?

Reply:
i think this should never happen anyways otherwise the game would have shown the "Enter %s" without any formatting which i think never happened